Library/ViewModels/LoginToutViewModelTests.swift

```diff
@@ -570,7 +570,13 @@ final class LoginToutViewModelTests: TestCase {
   }
 
   func testShowAppleErrorAlert_FetchUserEventError() {
-    withEnvironment(apiService: MockService(fetchUserError: .couldNotParseJSON)) {
+    let envelope = SignInWithAppleEnvelope.template
+      |> \.signInWithApple.apiAccessToken .~ "some_token"
+
+    withEnvironment(apiService: MockService(
+      fetchUserResult: .failure(.couldNotParseJSON),
+      signInWithAppleResult: .success(envelope)
```

Comment:
Interesting observation:  

this unit test broke after I updated the MockService's signInWithApple method to return `producer(from:)`. I had to manually include the argument to fix it whereas previously the omission of the parameter was being handled as an error. This is related to some of the MockService methods defaulting to a response of a template (Justin mentioned this was troublesome).

Reply:
Oh right I remember now, ok that's why some tests suddenly fail when the `response` is changed to `result`. I wasn't sure exactly why it was happening, but as you mention its because those values were previously set to `nil` by default. I had the same issue when converting over mock service responses and errors into results for the `UserStoredCards` fragment.